### ISSUE TITLE
Depth value scaling on test and train

### ISSUE THREAD

**Issue by JonathanCMitchell** [janivanecky/Depth-Estimation#14]
Question A) In your script in /dataset/test/crop.py. On lines 39-49 https://github.com/janivanecky/Depth-Estimation/blob/master/dataset/test/crop.py#L42  you have some logic that seems to be dividing the depth array by 65535.0. What is that value from? Is that the max depth value that the kinect sensor can get?

Question B) Also, are the ground truth depth maps for the gradient network scaled from [0, 255] or from [0, 10]?

Question C) If the ground truth depths for the gradient network (that are of shape 75, 54), are scaled from [0, 255], are you simply converting them to [0, 10] inside the loss function? Specifically the ScaleInvariantMeanSquaredError loss function inside /source/global_context_network/eval_depth.py.  https://github.com/janivanecky/Depth-Estimation/blob/master/eval_depth.py#L44. I don't really understand what is being passed to the LogDepth() function because if you divide by 10.0 then multiply by 10.0 isn't that the same thing as multiplying by 1?

Thanks!

**Comment by janivanecky** [comment]
Hi!

A.) Yes, kinect sensor outputs values as uint16, so that's the max value possible.
B.) Not sure anymore, sorrty :/
C.) LogDepth() function expects inputs scales to 0-1 range. 10.0 is maximum number of meters in the data, so assuming that network output specifies absolute value in meters, we first have to scale the output to 0-1 range (dividing by 10.0), convert to log-space (LogDepth() function), and convert back to absolute values (multiply by 10.0).

Sorry I'm not able to provide definite answers, it's been ~2 years since I worked on it.
